### ISSUE TITLE
A11y_SQL Azure Data Studio_Browse Preview_Azure tree item_Voiceover: voiceover is not announcing any command to access the "Sign in to azure"

### ISSUE THREAD

**Issue by ShaileshChudawkar** [microsoft/azuredatastudio#24343]
### GitHub Tags:
#A11yMAS; #A11ySev3; #A11yTCS; #Mac #BM_SQL Azure Data Studio_Mac_Aug2023; #SQL Azure Data Studio; #WCAG1.3.1; #VoiceOver; #Benchmark; 

### Environment Details:
Application Name: SQL Azure Data studio
OS: Mac
### Repro Steps:

1. Launch Azure Data Studio insiders application.
2. Navigate to Connections from left menu and in servers button and hit enter "New Connection" button.
3. Connections section will get open.
4. Tab to "Browse Preview" Tab item and hit enter.
5. Navigate to "Sign in to Azure " present in "Azure" tree item.
6. Verify voiceover is not announcing any command to access the "Sign in to azure"
### Actual Result:
Voiceover is not announcing any command to access the "sign in to azure".
### Expected Result:
Voiceover should announce command to access "sign in to azure". it helpful to add azure account.
### User Impact:
Screen reader user face difficulty if voiceover is not announcing command to access the control.
### 
https://github.com/microsoft/azuredatastudio/assets/143179527/42b0bc90-6274-4ca1-b584-df54bbd0d9d1
